### PR TITLE
Client tweet live updates

### PR DESCRIPTION
**Pull request by stephen-lt-sl**
Does live updates and stuff on the client pages; currently clients do not see deletes live, but they receive an object letting them know it's been deleted.

### CODE REVIEW

File: server/tweetSearch.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ module.exports = function(client) {
     var hashtags = ["#bristech", "#bristech2016"];
 
     function addTweetItem(tweets, tag) {
+        if (tweets.length === 0) {
+            return;
```
Comment (susanthomson) [inline server/tweetSearch.js:8]: Should this line return the tweetStore too? I can't face looking though the code to see where it's used, and it probably ignores the return value anyway but I guess consistency is good.

**Review by BRHoward — APPROVED** [review]

File: server/tweetSearch.js (hunk 1 above)
Comment (stephen-lt-sl) [reply server/tweetSearch.js:8]: The function as a whole doesn't return anything, it's just used to add to the tweet store

File: server/tweetSearch.js [hunk 2]
```diff
@@ -110,34 +113,41 @@ module.exports = function(client) {
         return tweetStore;
```
Comment (susanthomson) [inline server/tweetSearch.js:113]: Here it returns the tweetStore. I guess this line could return nothing too, I was just wondering if there was a reason to return it here but not if there are no new tweets added.

Comment (stephen-lt-sl) [reply server/tweetSearch.js:113]: That's actually a different function, GitHub just folds the unedited lines between the end of addTweetItem and getTweetStore out

Comment (susanthomson) [reply server/tweetSearch.js:113]: Yeah I just twigged :joy: :blush:

**Review by susanthomson — APPROVED** [review]

**Pull request merged**